csed2025/Students/info.txt

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Students",
+    "columns": [
+        "name",
+        "age",
+        "id"
+    ],
+    "primary_key": "id",
+    "Index_keys": [
+        "name"
+    ],
+    "Consistently": "Strong"
```

Comment:
Wht do you need this ? Is this for test ? If yes, please create a new folder call `test` and add all json and files related to the test